### PR TITLE
Fix minor issues in agents notebook when run using Colab

### LINKED ISSUE #146: Minor issues found when running `Build an agent with tool-calling superpowers` in Colab
I'm running the [Build an agent with tool-calling superpowers](https://huggingface.co/learn/cookbook/agents) notebook using [Colab](https://colab.research.google.com/github/huggingface/cookbook/blob/main/notebooks/en/agents.ipynb). While running the 1. 🏞️ Multimodal + 🌐 Web-browsing assistant section of the notebook in Colab, some minor issues arise.

1. Problem with langchain_community in `from langchain.agents import load_tools`. It generates the following error:

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/langchain/_api/module_import.py](https://localhost:8080/#) in import_by_name(name)
     68             try:
---> 69                 module = importlib.import_module(new_module)
     70             except ModuleNotFoundError as e:

14 frames
ModuleNotFoundError: No module named 'langchain_community'

The above exception was the direct cause of the following exception:

ModuleNotFoundError                       Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/langchain/_api/module_import.py](https://localhost:8080/#) in import_by_name(name)
     70             except ModuleNotFoundError as e:
     71                 if new_module.startswith("langchain_community"):
---> 72                     raise ModuleNotFoundError(
     73                         f"Module {new_module} not found. "
     74                         "Please install langchain-community to access this module. "

ModuleNotFoundError: Module langchain_community.agent_toolkits.load_tools not found. Please install langchain-community to access this module. You can install it using `pip install -U langchain-community`

```

2. When loading the SerpApi tool `search_tool = Tool.from_langchain(load_tools(["serpapi"])[0])`, it asks for the `SERPAPI_API_KEY`:

```
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
[<ipython-input-5-50699b13b9ec>](https://localhost:8080/#) in <cell line: 15>()
     13 
     14 
---> 15 search_tool = Tool.from_langchain(load_tools(["serpapi"])[0])
     16 
     17 

2 frames
[/usr/local/lib/python3.10/dist-packages/pydantic/v1/main.py](https://localhost:8080/#) in __init__(__pydantic_self__, **data)
    339         values, fields_set, validation_error = validate_model(__pydantic_self__.__class__, data)
    340         if validation_error:
--> 341             raise validation_error
    342         try:
    343             object_setattr(__pydantic_self__, '__dict__', values)

ValidationError: 1 validation error for SerpAPIWrapper
__root__
  Did not find serpapi_api_key, please add an environment variable `SERPAPI_API_KEY` which contains it, or pass `serpapi_api_key` as a named parameter. (type=value_error)

```

3. When calling the `InferenceClient` inside the `ReactCodeAgent`, it asks for access for the `HF access token`:

```
Please log in or use a HF access token

```

### PR DESCRIPTION
**Pull request by sergiopaniego**
# What does this PR do?

Fixes #146.

I've added the following changes to address the issues:

1. Installation of `langchain-community`.
2. A new cell for adding `SERPAPI_API_KEY`.
3. A cell for HF login.

## Who can review?

@merveenoyan and @stevhliu.

### CODE REVIEW

File: notebooks/en/agents.ipynb [hunk 1]
```diff
@@ -35,7 +35,7 @@
    "metadata": {},
```
Comment (merveenoyan) [inline notebooks/en/agents.ipynb:35]: <p>huggingface_hub comes with transformers and datasets so we could declutter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (merveenoyan) [reply notebooks/en/agents.ipynb:35]: ah no, you are supposed to remove `huggingface_hub` only

Comment (sergiopaniego) [reply notebooks/en/agents.ipynb:35]: My bad, I misunderstood the message. I've restored `datasets` and removed `huggingface_hub`.

**Review by stevhliu — APPROVED** [review]
Thanks for the fix!

**Pull request merged**